fineract-provider/src/test/resources/META-INF/context.xml

```diff
@@ -23,7 +23,6 @@
 
 <Context>
 	<Resource type="javax.sql.DataSource" name="jdbc/mifosplatform-tenants"
-		driverClassName="org.drizzle.jdbc.DrizzleDriver" url="jdbc:mysql:thin://localhost:3306/mifosplatform-tenants"
-		username="root" password="mysql" validationQuery="select 1" maxActive="10"
-		maxIdle="4" />
+		driverClassName="org.drizzle.jdbc.DrizzleDriver" jdbcUrl="jdbc:mysql:thin://localhost:3306/mifosplatform-tenants"
+		username="root" password="mysql" />
```

Comment:
as above, it should be possible to remove this entirely

Reply:
Seems to me that this resource file is used in build.gradle in operations such as "tomcatrunwar" etc..., if you see what I mean.

Reply:
well spotted.. but you should be able to either just remove that `configFile = file('src/test/resources/META-INF/context.xml')` in [build.gradle](https://github.com/apache/fineract/blob/develop/fineract-provider/build.gradle), because according to https://github.com/bmuschko/gradle-tomcat-plugin it should not be required; or if it's giving you trouble, you could just keep an empty [context.xml](https://github.com/apache/fineract/blob/develop/fineract-provider/src/test/resources/META-INF/context.xml) (but I don't think you need to; I expect you'll find that you can simply entirely remove that file, and any references to it that you'll find, as part of your work on this PR)